modules/stochastic_tools/test/tests/multiapps/commandline_control/master_wrong_num_params.i

```diff
@@ -0,0 +1,49 @@
+[Mesh]
```

Comment:
Can this input file be eliminated by using cli_args with one of the other files?